nodeList.json

```diff
@@ -0,0 +1,900 @@
+[
+   {
+      "name":"Willowdale Mesh Node",
+      "latitude":43.77031350236652,
+      "longitude":-79.41487752247622,
+      "onlineStatus":"Online",
+      "cardinalDirection":"Omni",
+      "floor":"0",
+      "meshHardware":"Unknown",
+      "IPV6Address":"Unknown",
+      "lastPing":"-1",
+      "rtt":"0"
+   },
+   {
+      "name":"Willowdale Mesh Node",
+      "latitude":43.76950388379755,
+      "longitude":-79.42292951393893,
+      "onlineStatus":"Online",
+      "cardinalDirection":"Omni",
+      "floor":"",
+      "meshHardware":"Unknown",
+      "IPV6Address":"Unknown",
+      "lastPing":"-1",
+      "rtt":"0"
+   },
+   {
+      "name":"E-Mesh",
+      "latitude":43.68306736430572,
+      "longitude":-79.52565314047547,
+      "onlineStatus":"Online",
+      "cardinalDirection":"North",
+      "floor":"13",
+      "meshHardware":"Orange Pi ZERO and Alfa Networks",
+      "IPV6Address":"fcd6:6334:552e:351b:39ae:9b95:f4ce:318a",
+      "lastPing":"13",
+      "rtt":"0"
+   },
+   {
+      "name":"benhylau",
+      "latitude":43.63518200000000,
+      "longitude":-79.40201470000000,
+      "onlineStatus":"Online",
+      "cardinalDirection":"South",
+      "floor":"1",
+      "meshHardware":"Unknown",
+      "IPV6Address":"h.groundupworks.com",
```

Comment:
Good question, what happens when the addresses changes though?

Honesty the schema has not been finalized yet i just wanted to start somewhere.